seqr/views/utils/terra_api_utils.py

```diff
@@ -67,6 +69,12 @@ def anvil_call(method, path, access_token, user=None, headers=None, root_url=Non
     url, headers = _get_call_args(path, headers, root_url)
     request_func = getattr(requests, method)
     headers.update({'Authorization': 'Bearer {}'.format(access_token)})
+    cache_key = 'terra_req__' + hashlib.md5((url + str(headers) + str(kwargs)).encode('utf-8')).hexdigest()
+    r_text = safe_redis_get_json(cache_key)
+    if r_text:
+        logger.info('Debug: Cache {} hit. {}'.format(cache_key, r_text))
```

Comment:
I don't think we should be using the info logger to log debug statements. Either this should be a logger.debug log that isn't actually logged in prod or we should take out the word "debug" from the message